### DISCUSSION TITLE
Does OpenJDK get updates via Windows Update?

### DISCUSSION THREAD

**Discussion by Some1NamedNate** [microsoft/openjdk#532] [Q&A] [answered]
I recently switched from Java 8 SE to Microsoft Build of OpenJDK. Immediately after installing MS OpenJDK, I began to wonder how it receives updates. Does it use Windows Update?

**Comment by karianna** [comment, marked as answer]
It does not, you should use the regular MSI installer mechanism to update :-).  A new versions/installer is released every 3 months.

**Comment by brunoborges** [comment]
If you install using `winget `, you will get the updates when you run `winget update`.

https://learn.microsoft.com/en-us/windows/package-manager/winget/
